src/platform/nrfconnect/wifi/WiFiManager.cpp

```diff
@@ -207,10 +207,29 @@ CHIP_ERROR WiFiManager::Scan(const ByteSpan & ssid, ScanResultCallback resultCal
     mWiFiState          = WIFI_STATE_SCANNING;
     mSsidFound          = false;
     mRecoveryArmed      = true;
+    // TODO Workaround for recovery mechanism to wait before the next scan request until the WiFi supplicant is not busy.
+    static bool workaroundDone;
```

Comment:
I know it's already `false` but maybe better initialize it explicitly anyway.

Reply:
Static is always initialized with 0 (false), isn't it? Or I can set it to false just before calling reboot if something goes wrong.